### PR TITLE
packages: mega bump

### PR DESCRIPTION
**Pull request by MilhouseVH**
This PR depends on the `pre_configure()` commit in #3531 for the `e2fsprogs` update.

This is a bump of pretty much all non-add-on (ie. "main") packages in the following directories:

* x11
* audio
* security
* tools
* sysutils
* devel
* print
* lang
* python
* network
* graphics
* multimedia
* compress
* debug
* databases
* web
* textproc

Packages not updated:

* `libunwind-1.3.1`: duplicate arm symbol, not sure why.
* `fuse-3.5.0`: `fuse-exfat` not compatible with `libfuse3`. No `fuse-exfat` progress in 2 years. Hmmm.
* `open-iscsi-2.0.877`: build broken, not entirely sure how to fix - skipped it for now
* `meson-0.50.1`: needs additional cross-compile options - to be investigated separately
* `v4l-utils-1.16.6`: needs Python3, brings new keymap format - will be bumped later in cycle

`image` and `all` add-on builds successful for RPi/RPi2/Generic/MiQi/A64.

### CODE REVIEW

File: packages/sysutils/open-vm-tools/package.mk [hunk 1]
```diff
@@ -24,7 +24,6 @@ PKG_CONFIGURE_OPTS_TARGET="--disable-docs \
                            --without-x \
                            --without-xerces \
                            --without-icu \
-                           --without-procps \
```
Comment (CvH) [inline packages/sysutils/open-vm-tools/package.mk]: no bump to `10.3.10` for open-vm-tools ?

Comment (MilhouseVH) [reply packages/sysutils/open-vm-tools/package.mk]: Added

**Comment by MilhouseVH** [comment]
I'm downgrading `fuse-exfat` from 1.3.0 to 1.2.8 as `fuse-exfat-1.3.0` has [compatibility issues](https://forum.kodi.tv/showthread.php?tid=343068&pid=2859150#pid2859150) with `fuse-2.9.9`.
```
● udevil-mount@-dev-sdb1.service - Udevil mount service
   Loaded: loaded (/usr/lib/systemd/system/udevil-mount@.service; static; vendor preset: disabled)
   Active: active (exited) since Wed 2019-06-05 19:09:19 BST; 1min 1s ago
  Process: 766 ExecStart=/usr/bin/udevil --mount /dev/sdb1 (code=exited, status=1/FAILURE)
  Process: 779 ExecStartPost=/usr/lib/samba/samba-autoshare (code=exited, status=0/SUCCESS)
 Main PID: 766 (code=exited, status=1/FAILURE)

Jun 05 19:09:19 NUC systemd[1]: Starting Udevil mount service...
Jun 05 19:09:19 NUC udevil[766]: fuse: mountpoint is not empty
Jun 05 19:09:19 NUC udevil[766]: fuse: if you are sure this is safe, use the 'nonempty' mount option
Jun 05 19:09:19 NUC udevil[766]: FUSE exfat 1.3.0
Jun 05 19:09:19 NUC udevil[766]: mount: mounting /dev/sdb1 on /media/LIBREELEC failed: No such device
Jun 05 19:09:19 NUC systemd[1]: Started Udevil mount service.
```

The issue is that `fuse`/`fuse-exfat` isn't getting the `nonempty` option even though it is configured for use by `udevil` - the `mount` command being used by `udevil` is:
```
/usr/bin/mount -v -t exfat -o nosuid,noexec,nodev,noatime,utf8,nonempty /dev/sdb1 /media/LIBREELEC
```
In this case `/usr/bin/mount` is the Busybox `mount` applet.

The top 5 commits [here](https://github.com/relan/exfat/commits/b80f452a39ee1e39e1ee552b9bf7d82b335f84a6) cause the breakage, which seems to be in preparation for `fuse-3.x`.

I did test https://github.com/relan/exfat/pull/105 but that didn't help.

Perhaps `fuse-exfat-1.3.0` now only works with `fuse-3.x` but `fuse-3.x` has it's own build-related challenges so this can probably wait until things straighten themselves out - I don't have the time/inclination to dig further into these `fuse` issues.

**Comment by antonlacon** [comment]
If this is still cooking, curl 7.65.1 released yesterday.

**Comment by MilhouseVH** [comment]
Thanks, have added curl-7.65.1 - looks like a fairly major bug-fix.

**Pull request merged**